sorts/binary_insertion_sort.py

```diff
@@ -54,8 +33,50 @@ def binary_insertion_sort(collection: list) -> list:
         collection[low] = val
     return collection
 
+def test_binary_insertion_sort():
```

Comment:
Are these tests being run in our `build` GitHub Action?  I think not.

Reply:
Yes, It does ran and passed all of them @cclauss